### DISCUSSION TITLE
Change appearance - resets stats?

### DISCUSSION THREAD

**Discussion by JokuScores** [boblord14/Dark-Souls-2-SotFS-CT-Bob-Edition#16] [Q&A]
Hi, reading through the readme and playing around with the table, I see that changing the appearance (character creation) will force a reset of the stats. On the readme it states not to directly change stats in case of a soft ban. Is there any way to change the appearance and return the character stats (I'm currently SL 50). Thank you for your hard work.

**Comment by boblord14** [comment]
“Directly changing stats” refers to modifying your stat values directly from the statistics tab, where you can just enter whatever value you want. The actual problem comes from your SL and stat points not matching up, which is an easy catch from the anticheat. 

What you can do and is more recommended is to just use addsoul(scripts-> build creation -> addsoul) and spawn your requisite number of souls to re-level your stats after you change your appearance. Just obviously make sure you make down what your stats and soul level were beforehand so you know what to set them to. 

Only downside is you’ll increase your soul memory, but that’s unavoidable and only an issue if you’re trying to directly do coop or whatnot.
